ui/raidboss/data/03-hw/raid/a8s.js

```diff
@@ -820,8 +854,8 @@
         'Vortexer': 'Wirbler',
       },
       'replaceText': {
-        '--orbs--': '--Kugeln--',
-        '--regulator check--': '--Dampfregler check--',
+        '--orbs--': '--kugeln--',
+        '--regulator check--': '--dampfregler check--',
```

Comment:
Can you check with @Akurosia if this is desired before changing it?

Reply:
it's because of the copy-paste because I had the idea to include it in the common file.
And in common, there is no capital for translations with dashes.
Must  i restored. @Akurosia